web/src/app/main/components/homepage/homepage.component.html

```diff
@@ -1,32 +1,128 @@
-        </mat-list-item>
-      </mat-list>
-      <mat-list fxFlex="50%" dense>
-        <h3 mat-subheader>Most Popular Projects</h3>
-        <mat-list-item *ngFor="let project of projects">
-          <img [src]="project.logoUrl" *ngIf="project.logoUrl" />
-          <mat-icon mat-list-icon [matBadge]="project.repositories.length" matBadgeColor="accent" *ngIf="!project.logoUrl && project.type === 'public'">lock_open</mat-icon>
-          <mat-icon mat-list-icon [matBadge]="project.repositories.length" matBadgeColor="accent" *ngIf="!project.logoUrl && project.type === 'private'">lock</mat-icon>
-          <h4 mat-line><a [routerLink]="['/projects', project.uid]">{{ project.title }}</a></h4>
-          <p mat-line>{{ project.description  || 'no description' }} | Views: {{project.views}} | Last Updated: {{ project.updatedOn.toDate() | timeAgo }}</p>
-        </mat-list-item>
-      </mat-list>
-    </mat-card-content>
-  </mat-card>
+<div class="home" fxLayout="column">
+  <div fxLayout="row" fxLayoutAlign="start center">
+    <div class="home__title">Homepage</div>
+  </div>
+
+      <dashboard-stats></dashboard-stats>
+  <div fxLayout="row" fxLayout.lt-lg="column" fxLayoutAlign.lt-lg="space-between scretch" fxLayoutAlign="space-between center">
+ 
+
+    <mat-card class="home__users-card" fxFlex="49">
+      <mat-card-header>
+        <div class="home__users-card__header">
+          <mat-icon svgIcon="home_page_icon"></mat-icon>
+        </div>
+        <mat-card-title class="home__users-card__title">
+          Active Users
+        </mat-card-title>
+        <mat-card-subtitle class="home__users-card__title__subtitle">
+          Most active users
+        </mat-card-subtitle>
+      </mat-card-header>
+
+      <mat-card-content class="home__users-card__content">
+        <table mat-table [dataSource]="users" class="home__users-card__content__table">
+          <ng-container matColumnDef="avatar">
+            <th mat-header-cell *matHeaderCellDef></th>
+            <td mat-cell *matCellDef="let element">
+              <img class="home__users-card__content__table__avatar" matListAvatar [src]="element.avatarUrl" [alt]="element.username" />
+            </td>
+          </ng-container>
+
+          <ng-container matColumnDef="title">
+            <th mat-header-cell *matHeaderCellDef>Title</th>
+            <td mat-cell *matCellDef="let element">
+              <div fxLayout.xs="column" fxLayoutAlign.xs="start start" class="home__users-card__content__table__title">
+                <a href="https://github.com/{{element.username}}">{{ element.name ? element.name : element.username }}</a>
+                <div *ngIf="isSmallScreen" fxLayout="row" fxLayoutAlign="start center" class="home__users-card__content__table__time">
+                  <mat-icon svgIcon="time_icon"></mat-icon>
+                  <span>
+                    {{ element.lastUpdated.toDate() | timeAgo }}
```

Comment:
This can go in single line `<span> {{ element.lastUpdated.toDate() | timeAgo }} </span>` . Same applied for all spans